django_tidb/features.py

```diff
@@ -423,7 +423,7 @@ def supports_default_in_lead_lag(self):
 
     @cached_property
     def supports_json_field(self):
-        return False
+        return self.connection.tidb_version >= (6, 3, )
```

Comment:
need >= 6.3? TiDB seems to support json field long ago.

Reply:
JSON is there since at least v4.0 but became GA in v6.3. Either this or a simple `True` would be fine with me.


![image](https://github.com/pingcap/django-tidb/assets/1272980/4f821249-debe-4840-84d0-2de57a323447)


https://docs.pingcap.com/tidb/stable/release-6.3.0#mysql-compatibility